sos/report/plugins/convert2rhel.py

```diff
@@ -23,6 +23,7 @@ def setup(self):
             "/var/log/convert2rhel/convert2rhel.log",
             "/var/log/convert2rhel/archive/convert2rhel-*.log",
             "/var/log/convert2rhel/rpm_va.log",
+            "/var/log/convert2rhel/convert2rhel-pre-conversion.*"
```

Comment:
minor nit, is it worth adding a comma at the end of the line?

Reply:
.. to prevent potential future error when one adds a new copy spec (well in fact some CI test error, only, if I am right).

Good point, I like prevention!

Reply:
it just means that when we add a new line, we're only changing 1 line, and not 2 lines in the same way here ;)